google-cloud-storage/src/com/google/cloud/tools/intellij/gcs/GcsToolWindowFactory.java

```diff
@@ -41,5 +41,6 @@ public void createToolWindowContent(@NotNull Project project, @NotNull ToolWindo
   @Override
   public void init(ToolWindow window) {
     window.setIcon(GoogleCloudStorageIcons.CLOUD_STORAGE);
+    window.setShowStripeButton(false);
```

Comment:
what is a "stripe" button? Is that the term they use for the side panel tabs?

Reply:
Yes it is.